ts/features/euCovidCert/analytics/index.ts

```diff
@@ -0,0 +1,57 @@
+import { getType } from "typesafe-actions";
+import { mixpanel } from "../../../mixpanel";
+import { Action } from "../../../store/actions/types";
+import { getNetworkErrorMessage } from "../../../utils/errors";
+import { euCovidCertificateGet } from "../store/actions";
+import {
+  EUCovidCertificateResponse,
+  isEuCovidCertificateSuccessResponse
+} from "../types/EUCovidCertificateResponse";
+
+export const trackEuCovidCertificateActions = (
+  mp: NonNullable<typeof mixpanel>
+) => (action: Action): Promise<void> => {
+  switch (action.type) {
+    case getType(euCovidCertificateGet.request):
+      return mp.track(action.type);
+    case getType(euCovidCertificateGet.success):
+      return mp.track(
+        action.type,
+        trackEuCovidCertificateGetSuccessResponse(action.payload)
+      );
+    case getType(euCovidCertificateGet.failure):
+      return mp.track(action.type, {
+        reason: getNetworkErrorMessage(action.payload)
+      });
+  }
+  return Promise.resolve();
+};
+
```

Comment:
Could we avoid to track if the FF is off?
```typescript
const emptyTracking = (_: NonNullable<typeof mixpanel>) => (__: Action) =>
  Promise.resolve();
export default euCovidCertificateEnabled ? trackEuCovidCertificateActions : emptyTracking;
```
if you're agree, we can remove export from `trackEuCovidCertificateActions`

Reply:
Changed in b5de891cc912e665b811840c14f59aa842446fab